src/components/workspace/chatbot/chatbot-button.component.tsx

```diff
@@ -49,7 +49,7 @@ const ChatbotButton = () => {
       {showMessage && (
         <div className={styles.messageContainer}>
           <span className={styles.chatbotText}>
-            Hello {provider}, Welcome to UgandaEMR Plus Chatbot
+            Hello {provider}, Welcome to iFHIS Chatbot
```

Comment:
can you make this configurable so that we dont have to hard code the system name

Reply:
This has been resolved @jabahum